src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -8111,6 +8111,13 @@ StorageReplicatedMergeTree::unlockSharedData(const IMergeTreeDataPart & part, co
         return std::make_pair(true, NameSet{});
     }
 
+    auto shared_id = getTableSharedID();
+    if (shared_id == toString(UUIDHelpers::Nil))
+    {
+        LOG_TRACE(log, "Part {} blobs can be rem)oved, because table {} comletely dropped", part.name, getStorageID().getNameForLogs());
```

Comment:
```suggestion
        LOG_TRACE(log, "Part {} blobs can be removed, because table {} comletely dropped", part.name, getStorageID().getNameForLogs());
```